### PR TITLE
Add additional markdown to readme

### PR DESCRIPTION
**Pull request by jameshosken**
#3 Add more markdown examples, including emphasis, links, tables, and quotes. Also link to some helpful resources.

### CODE REVIEW

**Comment by D31F43** [comment]
#3 I think it is better to add a reference when creating PR. 👍

**Comment by jameshosken** [comment]
#3 @NHibiki [EDIT - I am an idiot] I see! Thanks for the info.

File: README.md [hunk 1]
```diff
@@ -4,13 +4,31 @@
...
 A list
 * thing 1
 * thing 2
 * thing 3
+
+### Extra markdown
+
+__Fun__ _with_ [markdown](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)
+
+Wow! | Much | Organise
```
Comment (shiffman) [inline README.md:22]: Should we use International or American spelling?

File: README.md [hunk 2]
```diff
@@ -4,13 +4,31 @@
...
+--- | --- | ---
+Tables | are | fun
+:sparkles: | :rocket: | :metal:
+
+As spiderman's uncle said:
```
Comment (shiffman) [inline README.md]: Is Spiderman capitalized?

**Review by shiffman — CHANGES_REQUESTED** [review]
Let's decide on these two small nitpicks before merging.

File: README.md (hunk 1 above)
Comment (lunaog) [reply README.md:22]: international!

**Review by shiffman — APPROVED** [review]

**Comment by shiffman** [comment]
I think this is good to go now!

**Pull request merged**